tests/check_util.c

```diff
@@ -84,7 +84,7 @@ static int verify_name_allowed_from_string(const char* name,
 // Tests verify_name_allowed with empty config.
 // Nothing is permitted.
 START_TEST(test_verify_name_allowed_empty) {
-    const char allow_file[] = "";
+    const char allow_file[] = "# ";
```

Comment:
I think the test is supposed to make sure that it works with empty files and files with comments aren't empty. Would it maybe make sense to open `/dev/null` or something like that when `size` is 0?

(Also it isn't tested on FreeBSD here but it's tested in the avahi repository only: https://github.com/avahi/avahi/pull/768. Would it maybe make sense to add some basic action running `make check` and `make distcheck` (https://github.com/avahi/nss-mdns/issues/103) by analogy with what the current action does on Ubuntu?)

Reply:
> (Also it isn't tested on FreeBSD here but it's tested in the avahi repository only: https://github.com/avahi/avahi/pull/768. Would it maybe make sense to add some basic action running make check and make distcheck (https://github.com/avahi/nss-mdns/issues/103) by analogy with what the current action does on Ubuntu?)

I do think so, but right now I my free time is extremely limited.

Reply:
The kludge will be removed in https://github.com/avahi/avahi/pull/770